### PR TITLE
Adding video content moderation using image-based API

### PR DESCRIPTION
**Pull request by jianwamazon**
*Description of changes:*
Adding a new notebook demonstrating how to use Amazon Rekognition image-based API to conduct video content moderation.

### CODE REVIEW

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb [hunk 1]
```diff
@@ -0,0 +1,897 @@
...
+    "\n",
+    "Depends on your requirements on accuracy, cost, performance, and architecture complexity, you can choose either of the approaches that best suited for your use case. Please refer to this blog for the detailed [comparison of the two content moderation approaches](https://aws.amazon.com/blogs/machine-learning/how-to-decide-between-amazon-rekognition-image-and-video-api-for-video-moderation/).\n",
+    "\n",
+    "This lab will show you how to use [ffmpeg](https://ffmpeg.org/) to sample frames from video and store them as images, then send those images for content moderation using image moderatin API and show moderation results in json format"
```
Comment (ConnorKirk) [inline content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb]:
```suggestion
    "This lab will show you how to use [ffmpeg](https://ffmpeg.org/) to sample frames from video and store them as images, then send those images for content moderation using image moderation API and show moderation results in json format"
```

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb [hunk 2]
```diff
@@ -0,0 +1,897 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "103399ca-0e43-4199-aefa-c4b291f02d55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install ffmpeg\n",
+    "!conda install ffmpeg -y\n",
```
Comment (ConnorKirk) [inline content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb:76]: nit: I find it a bit odd that both conda and pip are used. Are both required or could you use just one?

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb [hunk 3]
```diff
@@ -0,0 +1,897 @@
...
+    "\n",
+    "# print(HOME_DIR)\n",
+    "# print(VIDEO_LOCATION)\n",
+    "\n",
+    "# Initializing environment variables\n",
+    "bucket_name = sm.Session().default_bucket()\n",
+    "region = boto3.session.Session().region_name\n",
+    "\n",
+    "os.environ[\"BUCKET\"] = bucket_name\n",
+    "os.environ[\"REGION\"] = region\n",
```
Comment (ConnorKirk) [inline content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb]: I can't find where these environment variables are used elsewhere in the file. Are they needed?

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb [hunk 4]
```diff
@@ -0,0 +1,897 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81c278bd-e099-4fd5-9ddc-4ac5d9b5ab70",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "prefix_to_be_deleted = list(set(list_temp_s3_prefix))\n",
```
Comment (ConnorKirk) [inline content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb]: I think you could simplify this. The `set` is iterable, so it should work the same
```suggestion
    "prefix_to_be_deleted = set(list_temp_s3_prefix)\n",
```

**Review by ConnorKirk — COMMENTED** [review]
Nice work! Overall it looks good. I added a few questions and comments

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb [hunk 5]
```diff
@@ -0,0 +1,897 @@
...
+    "for pf in prefix_to_be_deleted:\n",
+    "    for obj in data_bucket.objects.filter(Prefix=pf):\n",
+    "        s3.delete_object(Bucket=bucket_name, Key=obj.key)"
+   ]
+  }
+ ],
+ "metadata": {
+  "availableInstances": [
```
Comment (matteofigus) [inline content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb]: Do we need all this metadata for specifying all the EC2 instances? My preference would be to have the same metadata we have in the other files on the same folder to keep things consistent, unless these are actually mandatory for the lab to work.

**Review by matteofigus — CHANGES_REQUESTED** [review]

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb (hunk 2 above)
Comment (jianwamazon) [reply content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb:76]: I tried pip to install ffmpeg but was unsuccessful, that's why I switched to conda, should I mention this in the markdown area?

File: content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb (hunk 5 above)
Comment (jianwamazon) [reply content-moderation/02-video-moderation/03-content-moderation-video-with-image-api.ipynb]: I've removed EC2 instances in metadata section

**Review by matteofigus — APPROVED** [review]

**Review by ConnorKirk — APPROVED** [review]

**Pull request merged**